appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - go get -d github.com/golang/dep
   - cd "%GOPATH%\src\github.com\golang\dep"
   - git checkout -q v0.4.1
-  - go install -ldflags="-X main.version=v0.4.1" ./cmd/dep
+  - go install -ldflags="-X main.version=v0.5.0" ./cmd/dep
```

Comment:
Just realized, but doesn't this just set the version variable, but still build from the master branch?

Reply:
nvm, just bump the branch checked out on line 24